Makefile

```diff
@@ -25,9 +25,12 @@ API_TYPES_GENERATED:=$(API_TYPES_DIR)/$(API_ZZ_GENERATED).go
 API_GO_HEADER_FILE:=pkg/apis/header.go.txt
 
 # Container image-related variables
+CONTAINER_ENGINE ?= podman
```

Comment:
You are right, another work around I have seen on another repos is using the var IMAGE_BUILD_CMD 
it can be set to
IMAGE_BUILD_CMD=docker build
IMAGE_BUILD_CMD=buildah bud
IMAGE_BUILD_CMD=podman build

and leave the default to podman

Reply:
In that case this can probably be cleaned up even further by removing `USE_BUILDAH` completely.

Reply:
yup, if you like that approach I can make the proper changes

Reply:
I welcome everything that reduces code duplication. :)